.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Tests
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'docs/**'
+      - '*.rst'
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - 'docs/**'
+      - '*.rst'
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - {name: Linux, python: '3.11', os: ubuntu-latest, tox: py311}
+          - {name: Windows, python: '3.11', os: windows-latest, tox: py311}
+          - {name: Mac, python: '3.11', os: macos-latest, tox: py311}
+          - {name: '3.10', python: '3.10', os: ubuntu-latest, tox: py310}
+          - {name: '3.9', python: '3.9', os: ubuntu-latest, tox: py39}
+          - {name: '3.8', python: '3.8', os: ubuntu-latest, tox: py38}
+          - {name: '3.7', python: '3.7', os: ubuntu-latest, tox: py37}
+          - {name: '3.6', python: '3.6', os: ubuntu-latest, tox: py36}
+          - {name: '2.7', python: '2.7', os: ubuntu-latest, tox: py27}
+          - {name: Style, python: '3.10', os: ubuntu-latest, tox: stylecheck}
+    steps:
+      - uses: actions/checkout@v4
+      - name: update pip
+        run: |
+          pip install -U wheel
+          pip install -U setuptools
```

Comment:
Curious why you ran this as two separate commands rather than `pip install -U wheel setuptools`?

To be clear, this isn't a "please change" request, but simply curious to learn.